pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -181,12 +196,12 @@ func (p *k8sPlugin) Attest(ctx context.Context, req *workloadattestor.AttestRequ
 		// if the container was not located and there were no pods with
 		// uninitialized containers, then the search is over.
 		if !notAllContainersReady || attempt >= config.MaxPollAttempts {
-			log.Printf("container id %q not found (attempt %d of %d)", containerID, attempt, config.MaxPollAttempts)
+			p.log.Warn("container id not found; giving up", "container_id", containerID, "attempt", attempt)
```

Comment:
That can happen if the plugin system is broken or if the plugin messes up the signature or `SetLogger` so that it no longer conforms to `catalog.NeedsLogger`. However, due to the way built-ins are loaded for unit-testing (the plugin system is exercised), those types of problems should be caught by unit-tests. If we're worried about it, we could initialize the plugins with a discarding logger or something just so there are no panics just in case...